src/jsx/components/input_info.jsx

```diff
@@ -16,6 +16,9 @@ function InputInfo(props) {
     filename = is_full_path
       ? _.last(props.input_data["file name"].split("/"))
       : props.input_data["file name"];
+  var original_button = (<li className="dropdown-item">
+    <a href={window.location.href+"/original_file/original.fasta"}>Original file</a>
```

Comment:
I wonder if we can determine the original filename from the supplied json results